docs/podman-varlink.1.md

```diff
@@ -31,8 +31,16 @@ More will go here as the docs and api firm up.
     as well.
 -->
 
+## CONFIGURATION
+
+Users of the podman varlink service should enable the  io.projectatomic.podman.socket and io.projectatomic.podman.service.
```

Comment:
Yoy have a doubled space before `io.projectatomic.podman.socket`, and you probably want to backtick both `io....`.